docs/how-to/edit-live.md

```diff
@@ -56,11 +56,8 @@ host machine, include the following in your `values.yaml`:
 ```yaml
   initContainer:
     enabled: true
-    scratch:
-      root: path/to/scratch/area  # e.g. /dls_sw/<my_beamline>/software/blueapi/scratch
-      repositories: []
 ```
-
+When initContainer is enabled worker must have scratch config.
```

Comment:
Please amend this to add the scratch info to the main container config and explain that enabling the initContainer iterates over the scratch config, cloning and setting groups if enabled. If the initContainer is not enabled then the scratch area will still try to be loaded.